packages/go_router_builder/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+### 1.0.1
+
+- Changes the argument name of the auto-generated `go` method from `buildContext` to `context`.
```

Comment:
nit: you changed the `parameter`. The argument is the thing you pass in. 😄  – not a big deal

Reply:
@kevmoo 

Thanks, I fixed it and squashed CHANGELOG changes 👍